flux/values-okteto.yaml

```diff
@@ -73,6 +73,9 @@ flux:
     # Do not scan images that match these glob expressions
     excludeImage:
 
+    # Scan only images that match these glob expressions; if empty, all images are included
+    includeImage:
```

Comment:
@pchico83 for this, I'm thiking that it would be useful to have a `$REGISTRY` token that replaces this with the current registry name, so that by default, only private images in our registry are included.

Reply:
but even without setting it, the instructions in the blog post worked without any issues

Reply:
Should we just set it to `registry.cloud.okteto.net*` or something like that?